crates/ruma-common/src/events/room/message/emote.rs

```diff
@@ -0,0 +1,80 @@
+use serde::{Deserialize, Serialize};
+
+#[cfg(feature = "unstable-msc1767")]
+use crate::events::message::MessageContent;
+
+use super::FormattedBody;
```

Comment:
These should be one import group.

```suggestion
#[cfg(feature = "unstable-msc1767")]
use crate::events::message::MessageContent;
use super::FormattedBody;
```